data/net/minecraft/world/level/chunk/LevelChunk.mapping

```diff
@@ -44,6 +73,15 @@ CLASS net/minecraft/world/level/chunk/LevelChunk
 		ARG 2 compound
 	METHOD removeBlockEntity (Lnet/minecraft/core/BlockPos;)V
 		ARG 1 pos
+	METHOD removeBlockEntityTicker (Lnet/minecraft/core/BlockPos;)V
+		ARG 1 pos
+	METHOD removeGameEventListener (Lnet/minecraft/world/level/block/entity/BlockEntity;)V
+		ARG 1 blockEntity
+	METHOD replaceWithPacketData (Lnet/minecraft/world/level/chunk/ChunkBiomeContainer;Lnet/minecraft/network/FriendlyByteBuf;Lnet/minecraft/nbt/CompoundTag;Ljava/util/BitSet;)V
+		ARG 1 container
+		ARG 2 buffer
+		ARG 3 nbt
```

Comment:
iirc should be `tag` not `nbt` @sciwhiz12 correct me on this one

Reply:
`tag`, or possibly a more specific name in the context of the method (to identify what data the tag is holding, likd if its chunk data or metadata or something else).